example/index.js

```diff
@@ -1,10 +1,28 @@
 bopen.addEventListener('click', event => {
+
+  const IPFS_OPTIONS = {
+    EXPERIMENTAL: {
+      pubsub: true
+    },
+    preload: { enabled: false }
+  }
+
+  /**
+   *  This uses a local ipfs-js instance and local orbitdb cache instead of the
+   *  shared iframe storage available at 3box.io. Easier for testing and differing
+   *  configs. But in production you will get the best performance by using the
+   *  default iframe configuration.
+   */
+  const ipfs = new window.Ipfs(IPFS_OPTIONS)
+  const opts = { ipfs, iframeStore: false }
```

Comment:
I'm a bit worried about having this in the example as people will look at it and most likely copy from the example.

Reply:
Maybe we can have two different examples?

Reply:
im fine just using the iframe by default, this was just before we had it deployed anywhere.

Reply:
ah, I see!